docs/csharp/whats-new/csharp-8.md

```diff
@@ -488,6 +489,16 @@ Span<Coords<int>> coordinates = stackalloc[]
 
 For more information, see [Unmanaged types](../language-reference/builtin-types/unmanaged-types.md).
 
+## stackalloc in nested expressions
+
+Starting with C# 8.0, if the result of a [stackalloc](../language-reference/operators/stackalloc.md) expression is of the <xref:System.Span%601?displayProperty=nameWithType> or <xref:System.ReadOnlySpan%601?displayProperty=nameWithType> type, you can use the `stackalloc` expression in other expressions:
+
+```csharp
```

Comment:
I think this should be referenced from the samples repo ?

Reply:
To be consistent with all other snippets in this article, this one is embedded in the file. Furthermore, as I understand any snippet should be referenced only by one article at most (then one can safely edit the snippet according to the article context). So, referencing this one would require creating a new project for the snippets in this article in the samples repo. While that may be a good move (so, I agree with you @Youssef1313) , I think it's not in the scope of this PR.

Reply:
> any snippet should be referenced only by one article at most (then one can safely edit the snippet according to the article context).

Good point, I didn't know that.

Reply:
> any snippet should be referenced only by one article at most (then one can safely edit the snippet according to the article context).

This is something we'd like to change in the future, with better navigation between samples and articles. /cc @dend